File: VB/CustomWizardExample/Wizard/Views/IChoosePersonsPageView.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.DataAccess.Wizard.Views

Namespace CustomWizardExample.Wizard.Views
    Public Interface IChoosePersonsPageView
        Inherits IChooseDataSourceTypePageView

        Function ShowWarning() As Boolean
    End Interface
End Namespace

File: CS/CustomWizardExample/Wizard/Views/IChoosePersonsPageView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.DataAccess.Wizard.Views;

namespace CustomWizardExample.Wizard.Views {
    public interface IChoosePersonsPageView : IChooseDataSourceTypePageView {
        bool ShowWarning();
    }
}
